packages/tpetra/core/test/MultiVector/MultiVector_MicroBenchmark.cpp

```diff
@@ -99,26 +99,29 @@ namespace { // (anonymous)
       ::Tpetra::Details::ProfilingRegion region ("Tpetra Update Loop");
       for(size_t i=0; i<num_repeats; i++) {
         X.update(ONE,Y,ONE);
+        Kokkos::fence();
```

Comment:
This is fine for now, given that Trilinos currently requires `CUDA_LAUNCH_BLOCKING=1`.  In the future, if we ever relax that requirement, it would make sense to allow the individual `update` calls to be nonblocking, and move the fence after the `num_repeats` loop.